src/styles/main.scss

```diff
@@ -35,4 +35,7 @@ body {
 
 .bg--gray {
     background-color: #ccc;
+}
+.slick-list {
+    height: 450px !important;
```

Comment:
This is not a good idea, as your slides may be higher than `450px`. There is another reason why height on this carousel breaks. It's because you have `height: 100%` set on the other carousel at the top. Try to fix that.